rs_server/CADIP/openapi.py

```diff
@@ -0,0 +1,51 @@
+"""Utility module for openapi specification manipulation."""
+import argparse
+import json
+from pathlib import Path
+
+from fastapi.openapi.utils import get_openapi
+
+from rs_server.CADIP.cadip_backend import app
```

Comment:
This module was updated, use rather
`from rs_server_cadip.fastapi.cadip_app import app`